src/pages/projectSettings/tabs/EventLogTab/EventLogTab.tsx

```diff
@@ -38,11 +48,108 @@ export const EventLogTab: React.FC = () => {
     },
   });
 
+  type logEntry = {
+    timestamp: Date;
+    user: string;
+    before?: RepoEventSettingsFragment | ProjectEventSettingsFragment;
+    after?: RepoEventSettingsFragment | ProjectEventSettingsFragment;
+  };
+
+  const eventData = isRepo
+    ? repoEventData?.repoEvents?.eventLogEntries
+    : projectEventData?.projectEvents?.eventLogEntries;
+
   return (
-    <>
-      <div>This is the event log page for {identifier}</div>
-      <div> {JSON.stringify(projectEventData)}</div>
-      <div> {JSON.stringify(repoEventData)}</div>
-    </>
+    <div>
+      <div data-cy="event-log">
+        {((eventData as logEntry[]) || []).map(
+          ({ user, timestamp, before, after }) => (
+            /* @ts-expect-error */
+            <EventLogCard key={`event_log_${timestamp}`}>
+              <EventLogHeader user={user} timestamp={timestamp} />
+              <Table
+                data={getEventDiffLines(before, after)}
+                columns={[
+                  <TableHeader
+                    key="key"
+                    label={<span data-cy="key">Property</span>}
+                    sortBy={(datum: EventDiffLine) => datum?.key}
+                  />,
+                  <TableHeader
+                    key="before"
+                    label={<span data-cy="before">Before</span>}
+                    sortBy={(datum: EventDiffLine) => datum?.before}
+                  />,
+                  <TableHeader
+                    key="after"
+                    label={<span data-cy="after">After</span>}
+                    sortBy={(datum: EventDiffLine) => datum?.after}
+                  />,
+                ]}
+              >
+                {({ datum }) => (
+                  <StyledRow key={datum?.key} data-cy="event-log-table-row">
+                    <Cell>{datum?.key ? datum?.key : ""}</Cell>
+                    <Cell>{getEventValue(datum?.before)}</Cell>
+                    <Cell>{getEventValue(datum?.after)}</Cell>
+                  </StyledRow>
+                )}
+              </Table>
+            </EventLogCard>
+          )
+        )}
+      </div>
+    </div>
   );
 };
+
+interface Props {
+  timestamp: Date;
+  user: string;
+}
+
+export const EventLogHeader: React.VFC<Props> = ({ user, timestamp }) => (
+  <StyledHeader>
+    <div>
+      <H3>{getDateCopy(timestamp)}</H3>
+    </div>
+    <div> {user} </div>
+  </StyledHeader>
+);
+
+/* @ts-expect-error */
+const EventLogCard = styled(Card)`
+  margin-bottom: 48px;
+  padding: ${size.m};
+  width: 120%;
+`;
+
+const StyledRow = styled(Row)`
+  td:first-of-type {
+    width: 80%;
+    word-break: break-all;
+  }
+`;
```

Comment:
I ran into this issue too where I couldn't figure out how to directly style the `Cell`! Mohamed left a [comment](https://github.com/evergreen-ci/spruce/pull/1211#discussion_r855542333) on this other PR of mine, maybe it's worth trying this?